### DISCUSSION TITLE
[Theme] Allure theme for trilium

### DISCUSSION THREAD

**Discussion by JadeVane** [TriliumNext/Trilium#4206] [Show and tell]
Allure is a theme for trilium with a more beautiful interface and higher focus which means that just focus on what you need.

https://github.com/JadeVane/Allure

![screenshot](https://github.com/zadam/trilium/assets/34258753/c3db01a3-9563-4f63-a5a7-041a6e9aacb3)

**Comment by Nyar233** [comment]
看起来很漂亮
